config.yaml

```diff
@@ -20,7 +20,7 @@ tasks:
     task_description: "Clustering is the task of grouping similar documents together."
   PairClassification:
     icon: "🎭"
-    metric: ap
+    metric: max_ap
```

Comment:
won't this cause issues with external results? (@Muennighoff I believe we have discussed this before)

Reply:
I've changed refresh.py, but I'll leave comment open until @Muennighoff review

Reply:
I changed refresh.py, but I'll leave the comment open until @Muennighoff reviews it. But I rather left `max_ap` in config, because after https://github.com/embeddings-benchmark/mteb/issues/1037 there is no `ap` in results.